packages/config/src/projects/arbitrum.ts

```diff
@@ -185,10 +185,10 @@ export const arbitrum: Project = {
       contracts: {
         addresses: [
           {
-            address: '0x171a2624302775eF943f4f62E76fd22A6813d7c4',
+            address: '0xC234E41AE2cb00311956Aa7109fC801ae8c80941',
```

Comment:
```suggestion
            address: '0x171a2624302775eF943f4f62E76fd22A6813d7c4',
```
This address shouldn't have changed